src/Compilers/VisualBasic/Desktop/CommandLine/CommandLineParser.vb

```diff
@@ -1388,6 +1394,19 @@ lVbRuntimePlus:
                           End Function)
         End Function
 
+        Private Function ParseAnalyzerDependencies(name As String, value As String, diagnostics As IList(Of Diagnostic)) As IEnumerable(Of CommandLineAnalyzerDependency)
+            If String.IsNullOrEmpty(value) Then
+                ' TODO: localize <file_list>?
+                AddDiagnostic(diagnostics, ERRID.ERR_ArgumentRequired, name, ":<file_list>")
+                Return SpecializedCollections.EmptyEnumerable(Of CommandLineAnalyzerDependency)()
+            End If
+
+            Return ParseSeparatedPaths(value).
+                   Select(Function(path)
+                              Return New CommandLineAnalyzerDependency(path)
+                          End Function)
```

Comment:
I see. Looks like we use multiline lambda in ParseAnalyzers above - but not for other places such as ParseAssemblyReference. This is very minor - but using multi-line lambda here seems wasteful (3 lines instead of 1)...